engine/dlib/src/dlib/win32/mutex.h

```diff
@@ -3,22 +3,27 @@
 // Copyright 2009-2014 Ragnar Svensson, Christian Murray
 // Licensed under the Defold License version 1.0 (the "License"); you may not use
 // this file except in compliance with the License.
-// 
+//
 // You may obtain a copy of the License, together with FAQs at
 // https://www.defold.com/license
-// 
+//
 // Unless required by applicable law or agreed to in writing, software distributed
 // under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
 // CONDITIONS OF ANY KIND, either express or implied. See the License for the
 // specific language governing permissions and limitations under the License.
 
-#ifndef DM_FILE_DESCRIPTOR_PRIVATE_H
-#define DM_FILE_DESCRIPTOR_PRIVATE_H
+#ifndef DM_MUTEX_WIN32_H
```

Comment:
Erm, this is just a bit weird.
The "private" header was deleted.
But as long as the new file has the correct name + content, then I'm happy.